resources/download_fabricmanager.sh

```diff
@@ -23,8 +23,8 @@ mkdir -p /tmp/nvidia
 pushd /tmp/nvidia
 
 # Download Fabric Manager tarball
-OUTDIR=/out/nvidia-fabricmanager/$DRIVER_VERSION
-FABRICMANAGER_ARCHIVE="fabricmanager-linux-$ARCH_TYPE-$DRIVER_VERSION-archive"
-FABRICMANAGER_URL="https://developer.download.nvidia.com/compute/cuda/redist/fabricmanager/linux-$ARCH_TYPE/${FABRICMANAGER_ARCHIVE}.tar.xz"
-mkdir -p "$OUTDIR"
-wget --directory-prefix="${OUTDIR}" "${FABRICMANAGER_URL}"
\ No newline at end of file
+wget -O /tmp/keyring.deb https://developer.download.nvidia.com/compute/cuda/repos/debian12/x86_64/cuda-keyring_1.1-1_all.deb && dpkg -i /tmp/keyring.deb
+apt-get update
+#apt-get install -V nvidia-open-"$DRIVER_BRANCH"
+apt-get install -V nvidia-fabricmanager-"$DRIVER_BRANCH" nvidia-fabricmanager-dev-"$DRIVER_BRANCH"
```

Comment:
Is `nvidia-fabricmanager-dev` needed? I would have expected just `nvidia-fabricmanager`

Reply:
Will be removed.